examples/todo/src/controllers/todo.controller.ts

```diff
@@ -35,7 +44,18 @@ export class TodoController {
     @param.path.number('id') id: number,
     @param.query.boolean('items') items?: boolean,
   ): Promise<Todo> {
-    return await this.todoRepo.findById(id);
+    try {
```

Comment:
If `legacy-juggler-bridge` throws the specific error instead of the generic `Error`,
we won't need to `try-catch` here.

However, there is a catch: `legacy-juggler-bridge` needs to import `@loopback/rest`,
which would create a cyclic dependency.

Maybe `HttpErrors` should be a package of it's own?

Reply:
This one's for fixing https://github.com/strongloop/loopback-next/issues/1469.

Reply:
Does juggler throw errors with `status` or `statusCode` set for http? If so, we can check that pattern instead of forcing `HttpErrors`.

Reply:
BTW, `HttpErrors` is from its own module. See https://github.com/strongloop/loopback-next/blob/3731d0f1ee18a12b36a6ec3785f295008b50d339/packages/rest/src/index.ts#L36. `@loopback/rest` just re-export it.

Reply:
Juggler does not throw, it just returns `null`. I am exploring the possibilities at Juggler level, because in another case it throws a with `code` and `statusCode`.